templates/post.html

```diff
@@ -10,18 +10,18 @@ <h1>{{ title }}</h1>
   {# Post content #}
   <p style="font-size: small;">
     Posted on <time>{{ date|date:"Y-m-d" }}</time>
-    {% if author %}
+    {% if authors %}
     by
+    {% for author in authors|slice:":-2" %}
+    <span class="author">{% if author.url %}<a href="{{ author.url }}">{{ author.name }}</a>{% else %}{{ author.name }}{% endif %}{% if author.github %} (<a href="https://github.com/{{ author.github }}">@{{ author.github }}</a>){% endif %}</span>,
+    {% endfor %}
+    {% for author in authors|slice:"-2:-1" %}
+    <span class="author">{% if author.url %}<a href="{{ author.url }}">{{ author.name }}</a>{% else %}{{ author.name }}{% endif %}{% if author.github %} (<a href="https://github.com/{{ author.github }}">@{{ author.github }}</a>){% endif %}</span> and
```

Comment:
not very familiar with cactus, but is there any way to reduce the duplication using a macro or smth?

Reply:
Not really. We could put it into a separate template and include that here, but that also would also lead to superfluous newlines